src/components/Field/Field.js

```diff
@@ -13,45 +15,60 @@ function Field({ children, label, required, ...props }) {
       ({ props }) => props && props.required
     )
 
+  const id = useMemo(
+    () => (typeof children === 'function' ? `Field_${fieldId++}` : null),
+    [children]
+  )
+
+  const labelProps = id === null ? {} : { htmlFor: id }
+
   return (
-    <div
-      css={`
-        margin-bottom: ${3 * GU}px;
-      `}
-      {...props}
-    >
-      <label>
-        <div
-          css={`
-            display: flex;
-            margin-bottom: ${0.5 * GU}px;
-            color: ${theme.surfaceContentSecondary};
-            ${unselectable()};
-            ${textStyle('label2')};
-            line-height: ${2 * GU}px;
-          `}
-        >
-          {label}
-          {isRequired && (
-            <span
-              css={`
-                color: ${theme.accent};
-              `}
-              title="Required"
-            >
-              {'\u00a0*'}
-            </span>
-          )}
-        </div>
-        {children}
-      </label>
-    </div>
+    <Inside name="Field">
+      <div
+        css={`
+          margin-bottom: ${3 * GU}px;
+        `}
+        {...props}
+      >
+        <label {...labelProps}>
+          <div
+            css={`
+              display: flex;
+              align-items: center;
+              height: ${2 * GU}px;
+              margin-bottom: ${0.5 * GU}px;
+              color: ${theme.surfaceContentSecondary};
+              white-space: nowrap;
+              ${textStyle('label2')};
+              ${unselectable()};
```

Comment:
```suggestion
              ${unselectable};
```

I _think_ we settled on this earlier 😄